protobuf/lib/src/protobuf/field_set.dart

```diff
@@ -369,8 +369,10 @@ class _FieldSet {
     return _getDefault(_nonExtensionInfoByIndex(index)) as T;
   }
 
-  /// The implementation of a generated getter. Common case for submessages.
-  T _$getN<T>(int index) {
+  /// The implementation of a generated getter for a default value determined by
+  /// the field definition value. Common case for submessages. dynamic type
+  /// pushes the type check to the caller.
+  dynamic _$getND(int index) {
     var value = _values[index];
     if (value != null) return value as T;
```

Comment:
This is no longer generic in T. Travis also tells this. I guess we just want to remove the cast?

Reply:
Looks like a merge error.